.github/workflows/blui-ci.yml

```diff
@@ -183,14 +183,14 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'yarn'
-    - run: yarn --frozen-lockfile
-    - run: yarn build:lib
```

Comment:
Not sure, do we need build demo for doc site.

Reply:
yes, 
build:demo is NG Doc site. 
build:lib is the component library components